observability/grafana/DeterminedAPIServerMontoring.json

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
What do you think about adding a histogram visualization? I think it's a really nice way to view the latency of a single API.

Reply:
I didn't add a histogram because I like the consistency of all the plots being time series.

Let me know if you think it would be especially useful for end users and I can take a shot at adding one.

Reply:
I'm ok with that - makes sense. We can always revisit if there's a specific request or need for it!